src/backend/metrics/metrics.ts

```diff
@@ -161,6 +161,8 @@ export const trackEvent = async ({
   }
 }
 
+export const flushEvents = async () => rudderstack.flush()
```

Comment:
we have to flush the events in the launch handler so that `Game Closed` is sent on app close. Otherwise it is added to the cache and will wait a little while to see if there are other events that can be batch sent along with it